pom.xml

```diff
@@ -198,6 +198,15 @@
         <module>testing/trino-tests</module>
     </modules>
 
+    <!-- FIXME: remove before merging -->
+    <repositories>
+        <repository>
+            <id>iceberg-release-candidate</id>
+            <name>Iceberg Release Candidate</name>
+            <url>https://repository.apache.org/content/repositories/orgapacheiceberg-1121/</url>
+        </repository>
+    </repositories>
```

Comment:
reminder

Reply:
just FYI, Iceberg 1.2.0 artifacts are now officially available, so the `repositories` section can be removed

Reply:
Thanks, removed